app/models/github_user.rb

```diff
@@ -15,6 +15,10 @@ def github_avatar_url(size = 40)
     "#{avatar_url}&size=#{size}"
   end
 
+  def name(options = {})
+    GitHub::Errors.with_error_handling { @client.user(@id, options).name }
+  end
```

Comment:
This is redundant method. I made a change in https://github.com/education/classroom/pull/622 which allows us to create method based on the `attributes` method down below.